src/parser/statement.js

```diff
@@ -1071,7 +1090,7 @@ export default class StatementParser extends ExpressionParser {
         /* isConstructor */ false,
       );
       this.checkGetterSetterParamCount(method);
-    } else if (this.isLineTerminator()) {
+    } else if (this.isLineTerminator() || this.match(tt.comma)) {
```

Comment:
Does this mean methods can have trailing commas?

```js
class Foo {
  method() {
  },
}
```

I don't see a test for it anywhere.